simpledotorg/simple-server
# Silence Prometheus when testing

**Story card:** [sc-14065](https://app.shortcut.com/simpledotorg/story/14065/stop-incessant-prometheus-log-failures)

## Because

It's noisy, and clouds meaningful logs

## This addresses

Prometheus logging while testing

## Test instructions

Run a test and check you don't see logs like

```
2024-11-11T13:07:08.314425+00:00 app[web.1]: App 152 output: E, [2024-11-11T13:07:08.314216 #152] ERROR -- : Prometheus Exporter, failed to send message Connection refused - connect(2) for "localhost" port 9394
2024-11-11T13:07:08.814773+00:00 app[web.1]: App 152 output: E, [2024-11-11T13:07:08.814580 #152] ERROR -- : Prometheus Exporter, failed to send message Connection refused - connect(2) for "localhost" port 9394
```
